build.sbt

```diff
@@ -12,7 +12,10 @@ lazy val jsonrpc = project.in(file("jsonrpc"))
   .settings(
     name := "scalabrad-web-jsonrpc",
 
-    resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/",
+    resolvers ++= Seq(
+      "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/",
+      Resolver.bintrayRepo("maffoo", "maven")
```

Comment:
Should be able to add just the jcenter resolver (`Resolvers.jcenterRepo`) since the jsonquote packages are included in jcenter (same for scalabrad below).